app/web/pages/messages/[[...slug]].tsx

```diff
@@ -58,4 +58,4 @@ export default function LeaveReferencePage() {
   );
 }
 
-LeaveReferencePage.getLayout = appGetLayout();
+LeaveReferencePage.getLayout = appGetLayout({ variant: "full-width" });
```

Comment:
This was the magic one here to stop the weird scrolling, but then I had to add it to all the inner components. As it should be, I think.

Reply:
@bakeiro Do we want the footer on the desktop version though for this page? Or only no footer on the mobile version?

Reply:
Nevermind! (U can ignore my comment xd)

Reply:
I actually just did it as probably we don't want the footer on the messages page when I thought about it more...

Reply:
yeah that's the thing, I though about do we need the footer after log in? also I though, maybe we use the same solutions for the same problems, instead of different solutions for the same problem, but I don't know which is the difference between `full-width` and `noFooter`...

in case we need the footer in desktop I guess full-width? but since I dind't want to change more than needed I asked you to ignore my comment lol

Reply:
🤔🤔🤔 I'm still thinking, I don't think we need the footer at all in the messages page

![Screenshot_2024-10-29-10-58-27-78_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/user-attachments/assets/a5a186be-0709-47cd-8ed8-e28b231f676a)

Reply:
do we need it after the user logs in? we can discuss this in the today's meeting